tools/SchemaManager/Program.cs

```diff
@@ -63,7 +63,7 @@ private static ServiceProvider BuildServiceProvider(string[] args)
         // TODO: this won't work in OSS if the AuthenticationType is set to ManagedIdentity
```

Comment:
Is this True now?

Reply:
I think this just refers to the the use of connection strings in OSS? It would work today too if we used managed identities in app services.